app/controllers/ApiController.scala

```diff
@@ -62,12 +68,21 @@ case class ApiController @Inject() (
       val userGroups = userGroupService.allGroups.filter(
         _.organisationSetOrDeducted.exists(_.id == Organisation.franceServicesId)
       )
+      val franceServiceInstances = organisationService.franceServiceInfos.instances
+      val doNotMatchTheseEmails =
+        franceServiceInstances
+          .flatMap(_.contactMail)
+          .groupBy(identity)
+          .filter(_._2.length > 1)
+          .keys
+          .toList
```

Comment:
```suggestion
          .toSet
```
optimisation gratuite